tests/test_send_recv_two_workers.py

```diff
@@ -17,7 +17,7 @@
 # subprocess.check_call(cmd, shell=True)
 
 pynvml = pytest.importorskip("pynvml", reason="PYNVML not installed")
-ITERATIONS = 100
+ITERATIONS = 1
```

Comment:
Should we also do `from utils import ITERATIONS` here?

Reply:
for now, no.  I want to keep those test separated since test_send_recv_two_workers.py will eventually run in CI.  I'm also going to add CLI args to the debug test

Reply:
Ok cool. Honestly wasn't sure. So thanks for the clarification. 🙂